network/gossip.go

```diff
@@ -102,16 +104,16 @@ func (t *Topic) readLoop(sub *pubsub.Subscription, handler func(obj interface{},
 			continue
 		}
 
-		go func() {
+		go common.RetryForever(context.Background(), time.Second, func(context.Context) error {
```

Comment:
I don't think we need retry pattern here because some node can send some malicious  message that can not be unmarshalled

Reply:
Thanks, removed